data_collector:
  ros__parameters:
    joint_order_follower:
      - joint1
      - joint2
      - joint3
      - joint4
      - gripper_left_joint
    joint_order_leader:
      - joint1
      - joint2
      - joint3
      - joint4
      - gripper_left_joint